src/frontend/src/common/components/library/data_subview/components/DownloadModal/style.ts

```diff
@@ -133,3 +141,16 @@ export const StyledCheckbox = styled(Checkbox)`
     }
   }
 `;
+
+export const StyledCallout = styled(Callout)`
+  ${fontBodyXxs}
+  width: 100%;
+  max-width: 434px;
```

Comment:
Callout has a width set to 360 px, so I added the width: 100% to counteract that.  Without this max-width, width: 100% was making the whole modal bigger.  Open to alternatives here.